contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol

```diff
@@ -113,4 +113,14 @@ interface VRFCoordinatorV2Interface {
    * @param to - Where to send the remaining LINK to
    */
   function cancelSubscription(uint64 subId, address to) external;
+
+  /*
+   * @notice Check to see if there exists a request commitment consumers
+   * for all consumers and keyhashes for a given sub.
+   * @param subId where to send the funds
+   * @return exits true if outstanding requests
+   * @dev Looping is bounded to MAX_CONSUMERS*(number of keyhashes).
+   * @dev Used to disable subscription canceling while outstanding request are present.
```

Comment:
Should these be part of the implementation of this method rather than the interface?

Reply:
Ah yeah good catch, done.